app/views/users/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1> <%= @user.email %> </h1>
+
+<div>
+  <img src="<%= @user.avatar.url %>" height="200" width="200">
+</div>
+
+<div>
+  <h3>Your Reviews</h3>
+  <% @reviews.each do |review| %>
+    <li>Park Name: <%= review.park.name %><br>
+    Review: <%= review.body %></li>
```

Comment:
Typically each html line should have it's own tag. I would consider <p> for the second line if you don't want it to be bulleted as a list element.